sa/model.go

```diff
@@ -723,10 +724,26 @@ func incidentModelToPB(i incidentModel) sapb.Incident {
 	}
 }
 
-// incidentCertModel represents a row in an 'incident' table.
-type incidentCertModel struct {
+// incidentSerialModel represents a row in an 'incident' table. The fields must
+// match the order of the columns in the 'incident' table schema.
+type incidentSerialModel struct {
 	Serial         string    `db:"serial"`
 	RegistrationID int64     `db:"registrationID"`
 	OrderID        int64     `db:"orderID"`
 	LastNoticeSent time.Time `db:"lastNoticeSent"`
 }
+
+// columns returns a slice of incident table column names. This is used for
+// constructing SQL queries.
+func (i incidentSerialModel) columns() []string {
```

Comment:
```suggestion
func (ism incidentSerialModel) columns() []string {
```

Otherwise the `i` loop variable shadows the receiver of this method, which is surprising.

Reply:
Oh, good catch!